pkg/config/default.go

```diff
@@ -253,6 +253,7 @@ func defaultConfig() (*Config, error) {
 			Volumes:             attributedstring.Slice{},
 		},
 		Network: NetworkConfig{
+			DefaultFirewall:           "iptables",
```

Comment:
Keep the default unset/empty, this is important to not overwrite the default netavark detection.